highfidelity/hifi-content
# Hub pill has nonspatialized space effect

## Test Plan
1. Same as the pill script in hifi://limitless
2. Click on pill
3. You should hear gulp and space sound
4. try flying around; the space sound should not change volume
